templates/front/profile/follow.html.twig

```diff
@@ -22,17 +18,15 @@
                         <img src="{{ asset('build/images/defaultUser.png')}}" class="circle">
                     {% endif %}
                     <span class="title"><a href="{{ path('app_front_profile_show', {'username': user.follower.username}) }}">{{ user.follower.username }}</a></span>
-                    {% if user.follower.id in relationships %}
+                    {% if user.follower.id in followedsArray %}
                         <a class="btn btn-primary secondary-content" href="{{ path('app_front_profile_unfollow' , {'username': user.follower.username , "csrf_token" : csrf_token('follow')}) }}">Ne plus suivre</a>
-                    {% elseif user.follower.username == userConnected %}
-
-                    {% else %}
+                    {% elseif  (user.follower.username == userConnected) == false %}
```

Comment:
essaie de transformer `user.follower.username == userConnected` en `user.follower.username is not same as(userConnected)`